sdk/readme.md

```diff
@@ -8,4 +8,76 @@ See [./examples/account_age.rs](./examples/account_age.rs) for an example Axiom
 cargo run --example account_age -- --input data/account_age_input.json -k 12 -p <PROVIDER_URI> <CMD>
 ```
 
-where `PROVIDER_URI` is a JSON-RPC URL, and `CMD` is `mock`, `prove`, `keygen`, or `run`.
\ No newline at end of file
+where `PROVIDER_URI` is a JSON-RPC URL, and `CMD` is `mock`, `prove`, `keygen`, or `run`.
+
+
+## CLI 
+
+```Usage: account_age <COMMAND>
+
+Commands:
+  serve  Run a circuit proving server
+  run    Run keygen and real/mock proving
```

Comment:
hmm there is also `run` the command below. Is there a better word we can use here?

Reply:
changing the command below to `prove`